src/drive/web/modules/viewer/FilesViewer.jsx

```diff
@@ -52,11 +58,13 @@ const styleStatusBar = switcher => {
 const FilesViewer = ({ filesQuery, files, fileId, onClose, onChange }) => {
   useUpdateDocumentTitle(fileId)
   const [currentFile, setCurrentFile] = useState(null)
+  const [currentDecryptedFileURL, setcurrentDecryptedFileURL] = useState(null)
```

Comment:
```suggestion
  const [currentDecryptedFileURL, setCurrentDecryptedFileURL] = useState(null)
```
because it's a convention in the React hook: https://reactjs.org/docs/hooks-state.html

Reply:
Oh good catch